src/vs/platform/configuration/common/configurationRegistry.ts

```diff
@@ -11,6 +11,11 @@ import * as types from 'vs/base/common/types';
 import { IJSONContributionRegistry, Extensions as JSONExtensions } from 'vs/platform/jsonschemas/common/jsonContributionRegistry';
 import { IStringDictionary } from 'vs/base/common/collections';
 
+export enum EditPresentationTypes {
+	Multiline = 'multiline',
```

Comment:
These should keep the word "text", otherwise it feels not specific enough if you are looking at this with no context